lib/OpenQA/Schema/Result/Workers.pm

```diff
@@ -45,5 +46,47 @@ __PACKAGE__->has_many(properties => 'OpenQA::Schema::Result::WorkerProperties',
 # TODO
 # INSERT INTO workers (id, t_created) VALUES(0, datetime('now'));
 
+sub seen(;$) {
```

Comment:
I added a .perlcriticrc to ignore this - it seems to be very controversial as I found several HOWTOs to ignore it :)